utility_test.go

```diff
@@ -108,3 +108,49 @@ func TestNewDateTime(t *testing.T) {
 	assert.Equal(t, time.Duration(offset)*time.Second, date.GetUtcOffset().AsDuration())
 	t.Log("Offset:", offset, "Got:", date.GetUtcOffset().AsDuration().String())
 }
+
+func TestNewMoney(t *testing.T) {
```

Comment:
This might already be covered by the fuzzing tests, but please include tests for negative numbers and ensure that all requirements are met.

Reply:
Maybe it went unnoticed but there's already a test covering negative numbers.

Reply:
Yep, you're right. My quick scanning failed me. Thanks!